ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -186,6 +183,30 @@ open class NavigationBar: UINavigationBar {
         }
     }
 
+     /// The navigation bar's default leading content margin.
+    @objc public static let defaultContentLeadingMargin: CGFloat = 8
```

Comment:
Do the default values need to be public? Wouldn't they just be the value set to the properties themselves?

Reply:
Yes, they do need to be public.
The idea is to allow the customer to reset these properties to their default values.
For example, on iPad, we want to customize the nav bar's margins only when the screen is greater a certain width. Otherwise, we want to use the default values so we have to switch between a custom value and the default value.